editions/tw5.com/tiddlers/filters/examples/Lists.tid

```diff
@@ -0,0 +1,11 @@
+created: 20160114193046373
+modified: 20160115014200334
+tags:
+title: Lists
+type: application/x-tiddler-dictionary
+
+days: Mon Tue Wed Thu Fri Sat Sun
+months: Jan Feb Mar Apr May Jun Jul Aug Sep Oct Nov Dec
+dates: 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31
```

Comment:
As I saw those examples, I was immediately thinking: "Does this also work with the existing core, language specific sets?"

`$:/language/Date` prefixed tiddlers. .. If yes. some examples using those "build in" sets would be very valuable ;)